src/server.js

```diff
@@ -15,6 +16,8 @@ if (env.nodeEnv === 'development') {
   mongoose.set('debug', true);
 }
 
+app.use('/api/v1/users', userRoute);
```

Comment:
em đặt route ở vị trí đầu tiên này nó dừng lại trước khi vào i18n nên đa ngôn ngữ sẽ không có tách dụng => các route api đặt dưới api home '/' trên api not found '*'